pages/common/middleman.md

```diff
@@ -0,0 +1,28 @@
+# middleman
+
+> Static site generator written in Ruby.
+> More information: <https://middlemanapp.com/>.
+
+- Install the Middleman gem:
+
+`gem install middleman`
+
+- Create a new Middleman project:
+
+`middleman init "{{project_name}}"`
+
+- Start local server for current project on port 4567:
+
+`middleman server`
+
+- Start local server for current project on a specified port:
+
+`middleman server -p "{{port}}"`
+
+- Build your project to prepare to deploy:
```

Comment:
Perhaps we could avoid the use of `your`  here? Generally we try and avoid it in tldr pages. What about this instead?

```suggestion
- Build the project in the current directory to prepare for deployment:
```

Reply:
Fixed :+1: